README.md

```diff
@@ -5,7 +5,7 @@
 
 <!--
 
-TODO: Uncomment when php sdk published to public packagist:
+TODO: Uncomment when php sdk is published to public packagist:
```

Comment:
This is published now, no? @davehensleyns8

Reply:
@crfroehlich I've restored the original README.md from Dev though to stay on the side of caution.